.python-version

```diff
@@ -1 +1 @@
-3.10.11
+3.13.3
```

Comment:
We might have to stick to earlier version of Python, this one is not yet available for the latest `pyenv` (2.5.5).

Reply:
Have set it back to 3.12.6, which also matches converter service